app/src/main/java/co/nimblehq/compose/crypto/extension/ModifierExt.kt

```diff
@@ -0,0 +1,63 @@
+package co.nimblehq.compose.crypto.extension
+
+import android.graphics.BlurMaskFilter
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.drawBehind
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Paint
+import androidx.compose.ui.graphics.drawscope.drawIntoCanvas
+import androidx.compose.ui.graphics.toArgb
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import co.nimblehq.compose.crypto.ui.theme.Dimension.Dp0
+
+/**
+ * Original solution: https://github.com/wasim15185/jetpack-compose-crypto/blob/develop/app/src/main/java/co/nimblehq/compose/crypto/extension/BoxShadow.kt
+ * Draw a shadow affect under component
+ *
+ * borderRadius: Border radius of shadow
+ * blurRadius: Blur the shadow
+ * offsetY: Position of shadow according to Y axis
+ * offsetX: Position of shadow according to X axis
+ * spread: Squeeze or release the shadow of CardView
+ */
+fun Modifier.boxShadow(
+    color: Color = Color.Black,
+    borderRadius: Dp = Dp0,
+    blurRadius: Dp = Dp0,
+    offsetY: Dp = Dp0,
+    offsetX: Dp = Dp0,
+    spread: Float = 0f,
+    modifier: Modifier = Modifier,
+) = this.then(
+    modifier.drawBehind {
+        this.drawIntoCanvas {
+            val paint = Paint()
+            val frameworkPaint = paint.asFrameworkPaint()
+            val spreadPixel = spread.dp.toPx()
```

Comment:
to get rid of conversion

```suggestion
            val spreadPixel = spread.toPx()
```